lib/shipit/github_app.rb

```diff
@@ -52,7 +52,7 @@ def initialize(config)
       oauth = (@config[:oauth] || {}).with_indifferent_access
       @oauth_id = oauth[:id]
       @oauth_secret = oauth[:secret]
-      @oauth_teams = Array.wrap(oauth[:teams] || oauth[:teams])
+      @oauth_teams = Array.wrap(oauth[:teams])
```

Comment:
I'm not sure if this was accomplishing something or if it was just a typo. Maybe there's some obvious Ruby thing I'm missing here.

Reply:
Typo or brainfart most likely.